### PR TITLE
Bael 2509 v31

### PR DESCRIPTION
**Pull request by eaudet**
@echeyne Here's a new pull request. I have followed Josh comments and removed FileInputStream.

### CODE REVIEW

File: core-java-collections/src/test/java/com/baeldung/queueInterface/CustomBaeldungQueueUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package com.baeldung.queueinterface;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class CustomBaeldungQueueUnitTest {
```
Comment (echeyne) [inline core-java-collections/src/test/java/com/baeldung/queueInterface/CustomBaeldungQueueUnitTest.java]: Is this file relevant to your article?

**Comment by eaudet** [comment]
No. Perhaps it was changed when merging upstream.

**Comment by echeyne** [comment]
> No. Perhaps it was changed when merging upstream.

Hmm, that seems a bit odd. Can you please remove it? We only want file changes that are relevant to your article in this PR.

File: core-java-collections/src/test/java/com/baeldung/queueInterface/CustomBaeldungQueueUnitTest.java (hunk 1 above)
Comment (eaudet) [reply core-java-collections/src/test/java/com/baeldung/queueInterface/CustomBaeldungQueueUnitTest.java]: I will remove this commit. My bad...
